omnibus/config/software/datadog-agent-integrations-py2-dependencies.rb

```diff
@@ -27,6 +27,7 @@
   # add nfsiostat script
   dependency 'unixodbc'
   dependency 'freetds'  # needed for SQL Server integration
+  dependency 'msodbcsql18' # needed for SQL Server integration
```

Comment:
IS it needed given the sqlserver integration is now py3 only?

Reply:
good point. can i merge this PR then open a new pr to cleanup both `freetds` and `msodbcsql18`?

Reply:
I think it's ok to drop it in this PR but yes you're right we would need a second PR to cleanup `freetds`!